kview/src/main/java/dev/ikm/komet/kview/mvvm/view/pattern/PatternDetailsController.java

```diff
@@ -766,6 +769,8 @@ private void savePattern(ActionEvent actionEvent) {
         LOG.info(isValidSave ? "success" : "failed");
         updatePatternBanner();
         updateView();
+        actionEvent.consume();
```

Comment:
I don't think we should consume events (mouse events, etc) as a standard practice. 
The problem is that if we want to listen to this type of event somewhere up on the hierarchy by adding an event handler you won't be able to because the event was consumed. Also, there's no real advantage to do that in most cases.